data/sql/updates/pending_db_world/rev_1601315140580867400.sql

```diff
@@ -0,0 +1,125 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1601315140580867400');
+/*
+ * Zone: Azeroth (Skipped Part II)
+ * Update by Knindza | <www.azerothcore.org>
+ * Copyright (C) <www.shadowburn.net> & <www.lichbane.com>
+*/
+
+/* SMARTSCRIPT */
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=4011;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=4011 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(4011,0,0,0,0,0,100,0,5000,8000,12000,18000,11,744,32,0,0,0,0,2,0,0,0,0,0,0,0,'Young Pridewing - In Combat - Cast Poison');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=4263;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=4263 AND `source_type`=0;
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(4263,0,0,0,2,0,50,1,0,30,0,0,11,6536,0,0,0,0,0,1,0,0,0,0,0,0,0,'Deepmoss Hatchling - At 30% HP - Summon Deepmoss Matriarch');
+
+UPDATE `creature_template` SET `AIName`='SmartAI' WHERE `entry`=2172;
+DELETE FROM `smart_scripts` WHERE `entryorguid`=2172;
```

Comment:
this is dangerous! missing `source_type` can lead to deleting unwanted contents!